features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,30 @@
+Given("I am on the landing page") do
+  visit '/'
```

Comment:
I would prefer if you use the path helpers that Rails gives us. `'/'` vs `root_path`